$OpenBSD: patch-src_decoder_dec_mac_cpp,v 1.2 2016/11/03 13:07:06 jeremy Exp $

GetUTF16FromANSI is no longer under the CAPECharacterHelper namespace
in the current version of libmac.

--- src/decoder/dec_mac.cpp.orig	Thu Nov  3 13:39:04 2016
+++ src/decoder/dec_mac.cpp	Thu Nov  3 13:39:08 2016
@@ -174,7 +174,7 @@ mac_decoder_open(decoder_t * dec, char * filename) {
 
 
 	int ret = 0;
-        wchar_t * pUTF16 = CAPECharacterHelper::GetUTF16FromANSI(filename);
+        wchar_t * pUTF16 = GetUTF16FromANSI(filename);
         pdecompress = CreateIAPEDecompress(pUTF16, &ret);
         free(pUTF16);
 
